i2c/mcp9808_i2c/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Attaching a MCP9808 digital temperature sensor via I2C
+
+This example code shows how to interface the Raspberry Pi Pico to the MCP9808 digital temperature sensor board. 
+======
+This example reads the ambient temperature value each second from the sensor and sets upper, lower and critical limits for the temperature and checks if alerts need to be raised. The CONFIG register can also be used to check for an alert if the critical temperature is surpassed.

```

Comment:
"and critical limits for the temperature and checks if alerts need to be raised. The CONFIG register can also be used to check for an alert if the critical temperature is surpassed." - is the second sentence redundant here?

Reply:
Yeah agreed, would be wise to leave it out, I haven't covered that in the program